release-notes/Web_apps/Web_apps_Feature_Release_10.4/Web_apps_Feature_Release_10.4.9.md

```diff
@@ -55,11 +55,21 @@ If the component does not have this option enabled, the value will only be passe
 
 Because of a number of enhancements, from now on, all components that retrieve element/protocol parameters will do so more efficiently.
 
-#### Dashboards app & Low-Code Apps: All components now support zooming in/out using CTRL+Scroll [ID_40017]
+#### Dashboards app & Low-Code Apps: Several components now support zooming in/out using CTRL+Scroll [ID_40017]
```

Comment:
I've updated the release note itself, as I believe not all components support zooming in/out. Is this correct, @Robin-Debel? Below, I've added a list of the components that do offer zooming functionalities. Let me know if I've missed any. I will update the user guide accordingly if that is the case.

Reply:
You missed the timeline component, no overlay was added to this one because normal scrolling could also be applicable for scrolling up and down when items are grouped